Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:12-slim
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Copies your code file from your action repository to the filesystem path `/` of the container
+COPY entrypoint.sh /entrypoint.sh
+
+# Copy all trivy report data
+COPY *.json /data/
+
+# Mount data volume
+VOLUME /data
+
+# Install required packages
+RUN apt-get update && \
+    apt-get install -y tar ca-certificates gnupg curl --no-install-recommends && \
+    # Import Docker GPG key
+    install -m 0755 -d /etc/apt/keyrings && \
+    curl --retry 5 -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg && \
+    chmod a+r /etc/apt/keyrings/docker.gpg && \
+    # Add the Docker repository with the correct key ID
+    echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/debian $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+    tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+    # Install Docker
+    apt-get update && \
+    apt-get install -y docker-ce docker-ce-cli containerd.io --no-install-recommends
+
+# Install Copa
+RUN curl --retry 5 -fsSL -o copa.tar.gz https://github.com/project-copacetic/copacetic/releases/download/v0.2.0/copa_0.2.0_linux_amd64.tar.gz && \
```

Comment:
i thought i had a comment but i don't see it, can we parametrize copa version here

Reply:
reason is so that we don't have to release a new action version when there's a new copa version. we can default to `latest`

Reply:
yup, comment is here - https://github.com/project-copacetic/copa-action/pull/1#discussion_r1255063687